public/markdown/community-process.md

```diff
@@ -0,0 +1,95 @@
+# Community Process
+
+The purpose of this page is to make transparent the process by which decisions are made in Play Framework.  This is not a set of laws governing the Play project, nor is anything in this document new, it simply acknowledges the process that is already in place, and documents what it is.
+
+The goal of this page is to increase community contribution and sense of ownership over the Play project, through doing the following:
+
+* Making it clear and transparent to people who are new to the Play community how decisions are made, who makes those decisions, and how new people can attain any decision making responsibility.
+* Providing a concrete definition of the decision making process in Play, so that it can be referred to and improved upon, should the need arise.
+
+## Project ownership
+
+The Play project source code is owned and licensed by [Typesafe](https://www.typesafe.com).  Typesafe employees a team of developers who work on Play full time, and has the final say on both product decisions and technical decisions for the project.
+
+Though Typesafe wholely owns the Play project and has the final say in any decisions made, Typesafe does not own the Play community, nor does Typesafe take for granted its existence.  The Play community is arguably just as important, if not more important, than the Play project itself.  With over 400 community contributors to the core of Play, and hundreds more to the broader Play ecosystem, the Play community could easily survive without or in spite of Typesafe, if it ever came to that.
```

Comment:
Is that really how to spell it, or do you just love your daughter so much that you want to see her name everywhere?